polaris.shopify.com/src/components/TOC/TOC.module.scss

```diff
@@ -1,55 +1,34 @@
 .TOC {
-  font-size: var(--font-size-300);
-
-  > ul > li:not(:first-child) {
-    padding-top: 0.25rem;
-  }
-
   a {
-    color: inherit;
+    color: var(--p-color-text);
```

Comment:
This color token does not currently align with the Figma specs, but once color tokens are migrated to replace existing values with the experimental values, it will match.